apps/passport-server/src/services/telegramService.ts

```diff
@@ -193,6 +205,82 @@ export class TelegramService {
         this.rollbarService?.reportError(e);
       }
     });
+
+    // The "setup" command helps prospective event organizer to collect necessary information.
+    // Currently, this just prints the ID of the channel.
+    this.bot.command("setup", async (ctx) => {
+      if (ctx.chat?.type === "private") {
+        await ctx.reply(
+          "To get you started, can you please add me as an admin to the telegram channel associated with your event? Once you are done, please ping me again with /setup in the channel."
+        );
+        return;
+      }
+
+      const admins = await ctx.getChatAdministrators();
+      const isAdmin = admins.some(
+        (admin) => admin.user.id === this.bot.botInfo.id
+      );
+      if (!isAdmin) {
+        await ctx.reply(
+          "Please add me as an admin to the telegram channel associated with your event."
+        );
+        return;
+      }
+
+      await ctx.reply("Your telegram channel id is " + ctx.chat.id);
+    });
```

Comment:
nit: maybe /setup and /link should just be one command? similar to `doctor` command available in many CLI tools which just perform a list of checks to ensure things are working correctly and coach users how to fix them.

Reply:
Sounds good.

Reply:
Closed by b33df57dd879550d45e28add6934e37fc47fc062